src/amo/components/AddonMeta/styles.scss

```diff
@@ -0,0 +1,45 @@
+@import "~core/css/inc/mixins";
+@import "~amo/css/inc/vars";
+@import "~ui/css/vars";
+
+$background: #f8f9fa;
+
+.AddonMeta .Card-contents {
```

Comment:
It would and I tried but the content and layout is pretty different (right now). I wanted to land this so I admittedly rushed. I'd rather do that as a follow-up, if at all possible.